### DISCUSSION TITLE
Help With Student Developer Pack Application

### DISCUSSION THREAD

**Discussion by AceUtku** [community/community#63065] [GitHub Education]
### Hi everyone,

I'm a highschool student and i just wanted to get the benefits of Student Developer Pack myself since my financial situation now isnt the best. So I entered my schools name which was suprisingly on the list ( i live in Turkey and the city i live in is not really a major city), my mail, and my plan of using the pack. After i got the proof page i thought that the only thing i could reach at the moment was a document thats equivalent in English is something close to "Student Document". We use this document in almost anywhere we need to prove our studentships in Turkey. But after somewhile it was rejected for some reason. I thought that maybe some sort of error has occured so thats why it was rejected so i decided to try again but whenever i did, it said that i couldn't upload the same document. 
How can i solve this problem of mine? Can someone help or at least tell me how can i reach to people that can help please?

**Comment by humairshoukat** [comment]
Reach your closet GitHub Campus Expert. You can find this here https://education.github.com/experts
